admin/base/views.py

```diff
@@ -17,7 +17,7 @@ def home(request):
 
 
 def password_reset_done(request, **kwargs):
```

Comment:
I believe this view function can be removed entirely, because we're no longer using it in `admin.base.urls`.